lib/serviceAccessConfig/accessrulegenerator.py

```diff
@@ -151,12 +151,17 @@ def _get_allowed_client_ip_addrs(self):
             raise ServiceAccessGeneratorConfigError(error_msg)
 
         config_sections = ip_source_config.sections()
+        cidr_blocks = []
+        # add IPv4 ranges
         for section in config_sections:
-            public_ips = ip_source_config.get(section, 'public-ips')
-            cidr_blocks += public_ips
-            cidr_blocks += ','
-
-        cidr_blocks = cidr_blocks[:-1]  # remove trainling ,
+            subnets = ip_source_config.get(section, 'public-ips')
+            cidr_blocks.append(subnets)
+        # add IPv6 ranges
+        for section in config_sections:
```

Comment:
it seems it is doing the same for loop, why not doing inside the first `for section in config_sections` loop ?

Reply:
This keeps all the IPv4 addresses grouped together, and appends all the IPv6 addresses in a group after. In this way, the IPv4 section of the exported config looks the same; only the IPv6 is appended after. If I do it in one loop, they are intermingled, and that causes the exported config to look very different.